package.json

```diff
@@ -33,6 +33,7 @@
     "react-svg": "^15.1.7",
     "reflect-metadata": "^0.1.13",
     "typescript": "^4.7.2",
+    "uuid": "^9.0.0",
```

Comment:
@Saela Why did you add this dependency? We already have `uuidv4` and that's also what you're using in the service?